### ISSUE TITLE
Define Rule: Procedure must have one StudyIntervention that has AgentAdministration.

### ISSUE THREAD

**Issue by rbakercdisc1** [cdisc-org/ddf-core-poc#81]
The StudyDesign has a StudyIntervention.
This StudyIntervention is a Procedure.
The StudyIntervention has AgentAdministration.
AgentAdminstration has a name, label, and description.
The AgentAdministration has route, frequency, duration, and quantity.

How would this link to Biomedical Concepts? SDTM EC/EX BiomedicalConceptSurrogate? EC if blinded, EX if open-label. Are these pieces covered in the BC? It seems the USDM Entities noted will inform the BiomedicalConceptSurrogate for the study drug (aka AgentAdministration).

**Comment by DianeWold** [comment]
A StudyDesign has 0 to many StudyInterventions.  It is possible to have a study that does not have a StudyIntervention.
I think the description of the rule is sort of backward. There can be many Procedures in a StudyDesign, some of which may be StudyInterventions. 
I'm not certain the model is correct in saying that a StudyIntervention must have and AgentAdministration, unless we define AgentAdministration very broadly, to include, e.g., talk therapy, education, radiation, surgical procedures.

**Comment by rbakercdisc1** [comment]
If trialTypes  has any interventional, there must be a study design with at least one StudyIntervention of  AgentAdministration

if intervention is single group, it must have only one studyintervention....??

***Check scope of the model to make sure it is only interventional study with an agent that is administered.  @daveih what is the scope of the model?

**Comment by DianeWold** [comment]
In USDM v2.6, for StudyDesign a StudyIntervention is optional (0..1) while a StudyIntervention must have at least one AgentAdministration (1..*).
A StudyIntervention is a kind of Procedure, but since a StudyIntervention is optional, I don't think it's true that there must be at least one procedure that is a StudyIntervention.

**Comment by DianeWold** [comment]
@BSnoeijerCD I agree that if observational studies are in scope, then Procedures will not always include a StudyIntervention.  I also agree that some kinds of interventions will not have AgentAdministrations.
All that's left, then, is that if Study type (from SDTM codelist Study Type Response) is "INTERVENTIONAL" then there should be at least one StudyIntervention.
